src/libsyntax_pos/lib.rs

```diff
@@ -859,6 +859,17 @@ impl ExternalSource {
     }
 }
 
+#[derive(Debug)]
+pub struct OffsetOverflowError;
+
+impl fmt::Display for OffsetOverflowError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str("file offset does not fit in u32")
+    }
+}
```

Comment:
Do you expect this to be used by third parties?
(I'd rather not add dead code.)

Reply:
Makes sense, fixed!